Cartfile

```diff
@@ -1,2 +1,2 @@
-github "hyperwallet/hyperwallet-ios-sdk" "development"
+github "hyperwallet/hyperwallet-ios-sdk" "task/HW-61330-iOS-Update-UI-SDK-to-send-notification-for-authentication-error"
```

Comment:
I have approved Core SDK PR, coordinate with QA to get that merged 1st and then revert this change before merging this one

Reply:
I have updated the cart file